modules/machine-api-capability.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/cluster-capabilities.adoc
+
+:_content-type: REFERENCE
+[id="machine-api-capability_{context}"]
+= Machine API capability
+
+[discrete]
+== Purpose
+
+The `machine-api-operator`, `cluster-autoscaler-operator`, and `cluster-control-plane-machine-set-operator` Operators provide the features for the `MachineAPI` capability. You can only disable this capability if you install a cluster with user-provisioned infrastructure.
+
+The Machine API capability is responsible for all configuration and management of machines in the cluster. If you disable the Machine API capability during installation, you need to manage all machine-related tasks manually.
```

Comment:
Suggested reword of the first sentence, feel free to ignore if you like the OG better:
```suggestion
The Machine API capability is responsible for all of the machine configuration and management in the cluster. If you disable the Machine API capability during installation, you need to manage all machine-related tasks manually.
```